plugin/src/main/java/com/iridium/iridiumcore/utils/SkinUtils.java

```diff
@@ -22,8 +22,7 @@ public static UUID getUUID(String username) {
         try {
             if (uuidCache.containsKey(username.toLowerCase())) return uuidCache.get(username.toLowerCase());
 
-            String signature = getURLContent(
-                    "https://api.mojang.com/users/profiles/minecraft/" + username);
+            String signature = getURLContent("https://api.mojang.com/users/profiles/minecraft/" + username);
```

Comment:
do we need a signature.isEmpty here? cuz if the servers are down it will return an empty string

Reply:
Are you sure about that?